Core/src/org/sleuthkit/autopsy/casemodule/services/TagsManager.java

```diff
@@ -146,13 +147,41 @@ public static String getNotableTagLabel() {
         return tagDisplayNames;
     }
 
-    public static List<String> getNotableTagDisplayNames() {
+    /**
+     * Returns a list of the names of all Notable Tags.
+     * 
+     * @return Returns a list of notable tag definition names.
+     */
+    public static List<String> getNotableTagDisplayNames()  {
+        List<TagNameDefinition> tagNameDefinitionList = new ArrayList<>();
+        
+        tagNameDefinitionList.addAll(TagNameDefinition.getStandardTagNameDefinitions());
+        tagNameDefinitionList.addAll(TagNameDefinition.getTagNameDefinitions());
+        
         List<String> tagDisplayNames = new ArrayList<>();
-        for (TagNameDefinition tagDef : TagNameDefinition.getTagNameDefinitions()) {
+        for (TagNameDefinition tagDef : tagNameDefinitionList) {
             if (tagDef.getKnownStatus() == TskData.FileKnown.BAD) {
                 tagDisplayNames.add(tagDef.getDisplayName());
             }
         }
+        try {
+            TagsManager tagsManager = Case.getCurrentCaseThrows().getServices().getTagsManager();
+            TagSet categoryTagSet = tagsManager.getCategoryTagSet();
+            if(categoryTagSet != null) {
+                for(TagName tagName: categoryTagSet.getTagNames()) {
+                    if(tagName.getKnownStatus() == TskData.FileKnown.BAD) {
+                        tagDisplayNames.add(tagName.getDisplayName());
+                    }
+                }
+            }
+        } catch(TskCoreException ex) {
+            LOGGER.log(Level.WARNING, "Failed to retrieve category TagSet from DB");
+        } catch (NoCurrentCaseException ignored) {
+            /*
+             * No current case, nothing more to add to the set.
+             */
+        }
+
```

Comment:
What are the clients of this method? Given that it is a static method, there is an implication that it is called when there is no open case. If that is so, we may be breaking something that counted on all of the tag definitions to be in the Tag.properties file. Please look up the clients of this method and we'll move further discussion to Jira.